packages/halo-theme/src/custom-elements/ef-item.less

```diff
@@ -29,6 +29,7 @@
     }
     &[highlighted], &[focused]{
       color: @list-item-focus-text-color;
+      background-image: linear-gradient(@list-item-focus-background-color , @list-item-focus-background-color );
```

Comment:
Wrong formatting (extra spaces) and why does it need linear-gradient?

Reply:
Oh, thanks for reminding me about my mistakes, for linear-gradient you think I can use background-image: none instead, right?

Reply:
Can you use `background-color`?

Reply:
I think we can't do that because background-image is used for item selected display and it have their own color, so when selected item has highlighted or focused we need to change background-image or hide them.

<img width="683" alt="item selected highlighted" src="https://user-images.githubusercontent.com/102957966/168765923-25cef2aa-64d4-4211-8e77-e84ad785ea4e.png">

Reply:
Understood. Can you exclude `focused` and `highlighted` from `selected` state?

Reply:
Interesting,  exclude `highlighted` and `focused` is exist in `selected` scope, so i will  move backgroud-image scope to that

Reply:
updated